### ISSUE TITLE
NPE for skipped test

### ISSUE THREAD

**Issue by pskvortsov** [reportportal/agent-java-testNG#23]
Libs:
```
    rpTestngAgentVersion = '4.0.1'
    rpLog4jVersion = '4.0.0'
    rpLogbackVersion = '4.0.0'
    testngVersion = '6.14.2'
```

Steps to reproduce:
1. Create Example.java
```
import org.testng.Assert;
import org.testng.annotations.BeforeMethod;
import org.testng.annotations.Test;

public class Example {

    @BeforeMethod
    public void before() {
        Assert.assertTrue(false);
    }

    @Test
    public void test() {
        Assert.assertTrue(true);
    }
}
```
2. Add it to testng.xml
```
    <test name="Example test">
        <classes>
            <class name="Example" />
        </classes>
    </test>
```
3. Run via command line `./gradlew clean test`

Actual result: 
- test failed by NPE
```
java.lang.NullPointerException
	at com.epam.reportportal.testng.TestNGService.buildFinishTestMethodRq(TestNGService.java:324)
	at com.epam.reportportal.testng.TestNGService.finishTestMethod(TestNGService.java:155)
	at com.epam.reportportal.testng.BaseTestNGListener.onTestSkipped(BaseTestNGListener.java:103)
```

**Comment by pbortnik** [comment]
@pskvortsov 
Hello, i have some ideas where it could be a npe, but i cannot really reproduce the issue following these steps. 
So, could you please provide a little project where issue is reproduced?

**Comment by pbortnik** [comment]
[Duplicate of](https://github.com/reportportal/reportportal/issues/373).
Fixed in 4.0.2 version
